core/src/main/java/org/mule/runtime/core/internal/processor/AsyncDelegateMessageProcessor.java

```diff
@@ -60,25 +69,32 @@
 public class AsyncDelegateMessageProcessor extends AbstractMessageProcessorOwner
     implements Scope, Initialisable, Startable, Stoppable {
 
+  @Inject
+  private MuleContext muleContext;
   @Inject
   private SchedulerService schedulerService;
   @Inject
   private ConfigurationComponentLocator componentLocator;
 
   protected Logger logger = LoggerFactory.getLogger(getClass());
+
+  private boolean ownProcessingStrategy = false;
```

Comment:
Is there a use case for <async> having it's own processing strategy, rather than using flow processingStrategy?

Reply:
yes, for allow setting a `maxConcurrency` in the async (next PR to come)